tests/component_handlers/topic/test_proxy_wrapper.py

```diff
@@ -48,9 +49,10 @@ def setup(self, httpx_mock: HTTPXMock):
             status_code=200,
         )
         assert self.proxy_wrapper.host == HOST
-        assert self.proxy_wrapper.cluster_id == "cluster-1"
+        assert await self.proxy_wrapper.cluster_id == "cluster-1"  # type: ignore
```

Comment:
why is the type checker pragma necessary?

Reply:
because the `async_cached_property` decorator doesn't get detected as an async type

Reply:
with Pyright?

Reply:
yes